### PR TITLE
DX-622 added the csharp examples.

### PR DESCRIPTION
**Pull request by jchavez443**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-622.s3-website-us-east-1.amazonaws.com/

File: README.md [hunk 1]
```diff
@@ -81,7 +92,15 @@ curl --request POST \
 ```
 
 ```csharp
-// Coming Soon
+ApiCreateCallRequest callRequest = new ApiCreateCallRequest();
```
Comment (jmulford-bw) [inline README.md]: Can you add the proper `using` statements like you have for messaging?

**Review by jmulford-bw — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -47,7 +47,18 @@
 ```
 
 ```csharp
-// Coming Soon
+using BandwidthSdk.Standard.BandwidthMessaging;
+using BandwidthSdk.Standard.BandwidthMessaging.Controllers;
+
+APIController msgController =  new BandwidthMessagingClient(msgConfig).Client
```
Comment (ghost) [inline README.md]: For brevity sake, we can leave out initialization from the Readme snips

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: For brevity sake, we can leave out initialization from the Readme snips

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-622.s3-website-us-east-1.amazonaws.com/

File: README.md [hunk 3]
```diff
@@ -47,7 +47,17 @@
...
+using Bandwidth.Standard.Messaging.Models;
+using Bandwidth.Standard.Messaging.Controllers;
+
+MessageRequest msgRequest = new MessageRequest();
+msgRequest.ApplicationId = applicationId;
+msgRequest.From = "+18888888888";
+msgRequest.To = new string[1] {"9199199999"};
+msgRequest.Text = "The quick brown fox jumps over a lazy dog.";
+
+msgController.CreateMessage(msgUserId, msgRequest);
```
Comment (jmulford-bw) [inline README.md:60]: For consistency can you either change the user id to look like the example on line 101, or change the one on line 101 to look like this?

**Review by jmulford-bw — CHANGES_REQUESTED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-622.s3-website-us-east-1.amazonaws.com/

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md:60]: `msgController.CreateMessage(accountId, msgRequest);`

**Review by jmulford-bw — APPROVED** [review]

**Pull request merged**